runtime/browser/demo/recipes.manifest

```diff
@@ -26,12 +26,24 @@ recipe
 
 # Create shortlist with [product, ...] and suggest similar products from [person]'s wish list
 recipe
-  Chooser.choices -> Recommend.recommendations
-  Chooser.resultList -> ShowProducts.list
-  Chooser.resultList -> Recommend.known
+  # TODO: replace with this once connection constraints are fixed.
```

Comment:
@shans I had to disable the connection constraints here for now, since we were still resolving the results via the resolve by name strategy.